qemu/tests/cfg/blockdev_inc_backup_bitmap_size.cfg

```diff
@@ -33,3 +33,7 @@
             image_cluster_size_data1 = 512
             granularity_list = 512 1K 2K 4K 8K 16K 32K 64K 128K 256K 512K
             granularity_list += ' 1M 2M 4M 8M 16M 32M 64M 128M 256M 512M 1G 2G'
+    iscsi_direct:
+        lun_data1 = 1
+        enable_iscsi_full = no
+        image_raw_device_full = no
```

Comment:
"9     only filesystem" specified in this cfg file, so we need to decide if to support iscsi.

Reply:
Have remove this limitation "only filesystem".